compiler/x/codegen/OMRTreeEvaluator.hpp

```diff
@@ -1,452 +1,451 @@
-   static TR::Register *commonFPRemEvaluator( TR::Node *node, TR::CodeGenerator *cg, bool isDouble);
-   static TR::Register *generateLEAForLoadAddr(TR::Node *node, TR::MemoryReference *memRef, TR::SymbolReference *symRef,TR::CodeGenerator *cg, bool isInternalPointer);
-
-   static bool constNodeValueIs32BitSigned(TR::Node *node, intptrj_t *value, TR::CodeGenerator *cg);
-
-   static TR::Register *intOrLongClobberEvaluate(TR::Node *node, bool nodeIs64Bit, TR::CodeGenerator *cg);
-
-   enum
-      {
-      bandOpPackage = 0,
-      candOpPackage = 1,
-      sandOpPackage = 1,
-      iandOpPackage = 2,
-      landOpPackage = 3,
-      borOpPackage  = 4,
-      corOpPackage  = 5,
-      sorOpPackage  = 5,
-      iorOpPackage  = 6,
-      lorOpPackage  = 7,
-      bxorOpPackage = 8,
-      cxorOpPackage = 9,
-      sxorOpPackage = 9,
-      ixorOpPackage = 10,
-      lxorOpPackage = 11,
-
-      numLogicalOpPackages,
-      lastOpPackage = numLogicalOpPackages-1
-      };
-
-   static TR_X86OpCodes _logicalOpPackage[numLogicalOpPackages][numLogicalOpForms];
-
-   static bool generateIAddOrSubForOverflowCheck(TR::Node *compareNode, TR::CodeGenerator *cg);
-   static bool generateLAddOrSubForOverflowCheck(TR::Node *compareNode, TR::CodeGenerator *cg);
-
-   static TR::Register *SSE2ArraycmpEvaluator(TR::Node *node, TR::CodeGenerator *cg);
-   static TR::Register *SSE2ArraycmpLenEvaluator(TR::Node *node, TR::CodeGenerator *cg);
-
-   static bool stopUsingCopyRegAddr(TR::Node *node, TR::Register*& reg, TR::CodeGenerator *cg);
-   static bool stopUsingCopyRegInteger(TR::Node *node, TR::Register*& reg, TR::CodeGenerator *cg);
-
-   };
-
-}
-}
-
-#endif
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#ifndef OMR_X86_TREE_EVALUATOR_INCL
+#define OMR_X86_TREE_EVALUATOR_INCL

```

Comment:
There seems to be some whitespace changes in this file making it seem like every line was changed. Was this intentional?

Reply:
This is an end of line issue. I will fix them.

Reply:
> This is an end of line issue. I will fix them.

Fixed in 01d3018 .